app/views/recruits/_form.html.slim

```diff
@@ -24,3 +24,8 @@
   .field.recruit_form-btn
     .actions
       = f.submit "Save", class: "btn"
+
+= javascript_include_tag "https://maps.googleapis.com/maps/api/js?libraries=places&key=#{Rails.application.credentials.dig(:GOOGLE_API_SERVER_KEY)}"
+= javascript_pack_tag 'autocomplete'
```

Comment:
どちらもフォーム画面で読み込まれないとエラーになります？
autocompleteの中でフォームでしか参照できないDOMの要素を参照してたりしませんかね？
if文かイベントフック(フォームの要素が歩かないか判断してから処理を行う)で対応できないですかね。

Reply:
>どちらもフォーム画面で読み込まれないとエラーになります？

javascript_include_tagをapplication.html.slimに書くと下記エラーがでて
```
You have included the Google Maps JavaScript API multiple times on this page. This may cause unexpected errors.
```

javascript_pack_tagをapplication.jsに書くとフォームでautocompleteが機能しないです。

>if文かイベントフックで対応

ちょっと見てます。

Reply:
> javascript_include_tagをapplication.html.slimに書くと下記エラーがでて

Slackで回答しました。

> javascript_pack_tagをapplication.jsに書くとフォームでautocompleteが機能しないです。

`new google.maps.places` あたりでエラーが出てるわけでもなく、使えない状態と言うことですか？ なんとなく、google js が読み込めてなくてundefindとかになってないかと思ったのですが。

Reply:
Slackで返信しましたように一旦mergeして後のissueに回します！